package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "js-payments-card",
+  "name": "@securetrading/js-payments-card",
   "version": "1.0.0",
   "description": "TypeScript Payment Card",
-  "private": true,
+  "private": false,
```

Comment:
@james-simpson-ST These are couple of changes which I had to add before upload the package.

Reply:
We should first merge this PR and then release a new version (I'll synchronize rest of branches after merge).

Reply:
Makes sense :)